### PR TITLE
wrf-io: added NOAA software maintainers, added openmp variant

### PR DESCRIPTION
**Pull request by edwardhartnett**
added NOAA software maintainers to maintainer list, added openmp varient to wrf-io

### CODE REVIEW

**Review by Hang-Lei-NOAA — DISMISSED** [review]

**Review by t-brown — DISMISSED** [review]

File: var/spack/repos/builtin/packages/wrf-io/package.py [hunk 1]
```diff
@@ -9,14 +9,21 @@
...
     homepage = "https://noaa-emc.github.io/NCEPLIBS-wrf_io"
     url      = "https://github.com/NOAA-EMC/NCEPLIBS-wrf_io/archive/refs/tags/v1.2.0.tar.gz"
 
-    maintainers = ['t-brown']
+    maintainers = ['t-brown', 'kgerheiser', 'Hang-Lei-NOAA', 'edwardhartnett']
 
     version('1.2.0', sha256='000cf5294a2c68460085258186e1f36c86d3d0d9c433aa969a0f92736b745617')
 
-    depends_on('netcdf-c')
+    variant('openmp', default=False, description='Enable multithreading with OpenMP')
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/wrf-io/package.py:23]: Why remove `NetCDF-c` as a dependency?  It is still being listed as a prerequisite in the documentation.

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/wrf-io/package.py:23]: By documentation I mean the `README.md` at https://github.com/NOAA-EMC/NCEPLIBS-wrf_io.

Comment (edwardhartnett) [reply var/spack/repos/builtin/packages/wrf-io/package.py:23]: Because it is already a requirement of netcdf-fortran.

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/wrf-io/package.py:23]: Okay.  I was expressing a concern about the inconsistency w.r.t. the above documentation.

Comment (t-brown) [reply var/spack/repos/builtin/packages/wrf-io/package.py:23]: Which was the exact reason I had it listed. @edwardhartnett  would you like me to open an upstream PR with `s/netcdf C/netcdf Fortran/` for the documentation?

Comment (edwardhartnett) [reply var/spack/repos/builtin/packages/wrf-io/package.py:23]: @t-brown no I don't think that's a good idea. Some package require netcdf-c, some netcdf-c and netcdf-fortran.

Comment (t-brown) [reply var/spack/repos/builtin/packages/wrf-io/package.py:23]: I'm not following. You've removed the `netcdf C` dependency in this package, in looking at the `CMakeLists.txt` there is only a requirement for Fortran.

```
find_package(NetCDF REQUIRED COMPONENTS Fortran)
```

I'm agreeing with  @tldahlgren that this is inconsistent and I would like to clear that up.

Comment (edwardhartnett) [reply var/spack/repos/builtin/packages/wrf-io/package.py:23]: Whenever netcdf-fortran is used, netcdf-c is required. So the netcdf-c dependency is still there. I will return it to the package file...

**Review by tldahlgren — APPROVED** [review]
LGTM.  Thanks!

**Comment by edwardhartnett** [comment]
Looks like some network timeout:

`ERROR: Job failed (system failure): error dialing backend: read tcp 192.168.63.57:56638->192.168.26.103:10250: read: connection reset by peer`

**Comment by tldahlgren** [comment]
@spackbot run pipeline

**Comment by edwardhartnett** [comment]
Same timeout. Can I start the pipeline? 

@spackbot run pipeline

**Comment by tldahlgren** [comment]
> Same timeout. Can I start the pipeline?
> 
> @spackbot run pipeline

I *think* you can but it may have to be the only contents of the comment.

**Comment by tldahlgren** [comment]
Luckily the pipeline is optional right now (until the connectivity issues are addressed).  So I just needed to disable auto-merge before I could perform the merge operation.

**Pull request merged**